pkg/core/mempool/mempool.go

```diff
@@ -429,94 +424,6 @@ func (m Mempool) processGetMempoolTxsRequest(r rpcbus.Request) (interface{}, err
 	return outputTxs, err
 }
 
-// uType translates the node.TxType into transactions.TxType.
-func uType(t node.TxType) (transactions.TxType, error) {
```

Comment:
Should we still check the tx type? The ExecuteStateTransition return 2 different types

Reply:
Not sure what you mean. This `uType` was used only for converting  `node.TxType` (the one to be deprecated) to transactions.TxType. Nothing relevant here to ExecuteStateTransition.